tests/python/gaia-ui-tests/gaiatest/tests/functional/clock/test_clock_run_stopwatch_laps.py

```diff
@@ -0,0 +1,54 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest.apps.clock.app import Clock
+
+class TestClockRunStopWatch(GaiaTestCase):
+
+    def setUp(self):
+        GaiaTestCase.setUp(self)
+
+        self.clock = Clock(self.marionette)
+        self.clock.launch()
+
+    def test_click_run_stopwatch_laps(self):
+
+        # Switch to the stopwatch view.
+        stopwatch_view = self.clock.switch_view("stopwatch")
+
+        # Verify that the stopwatch time is at 0.
+        assert stopwatch_view.current_time == '00:00.00'
```

Comment:
`GaiaTestCase` inherits from [unittest.TestCase](https://docs.python.org/2/library/unittest.html), so you can use self.assertEqual(stopwatch_view.current_time, '00:00.00'). If the test fails at that place, the error message will be clearer. Python will show the string difference. With the current assert, Python will just say: AssertionError with the stacktrace.